core/gui/src/app/workspace/service/workflow-result-export/workflow-result-export.service.ts

```diff
@@ -127,14 +129,17 @@ export class WorkflowResultExportService {
   /**
    * export the workflow execution result according the export type
    */
-  exportWorkflowExecutionResult(
+  exportWorkflowExecutionResultToLocal(
```

Comment:
since this function can also export the result to the dataset, the function name should be `exportWorkflowExecutionResult`